examples/transform.js

```diff
@@ -1,6 +1,6 @@
 const write = require('csv-write-stream')
-const through = require('through2')
```

Comment:
there's value in showing how to use this with `through2` since that's still widely used. renaming the original `transform.js` to `transform-through2.js` and then modifying this file would've been a better route.

Reply:
it doesn't, and we should. however, `through2` is still getting 4 million downloads a month and is still widely used. removing its usage is removing value for users. that's why I suggested renaming the file in its original state to express that it's a through2 example. that way we keep both.